src/AST-Core/RBMessageNode.class.st

```diff
@@ -195,19 +195,19 @@ RBMessageNode >> isUnary [
 
 { #category : #accessing }
 RBMessageNode >> keywords [
-	^ selector keywords.
+	^ self selector keywords.
 ]
 
 { #category : #accessing }
 RBMessageNode >> keywordsIntervals [
-	^ selector keywords
+	^ self selector keywords
 		with: self keywordsPositions 
 		collect: [ :keyword :start | start to: (start == 0 ifTrue: [ -1 ] ifFalse: [ start + keyword size - 1]) ].
 ]
 
 { #category : #accessing }
 RBMessageNode >> keywordsPositions [
-	^keywordsPositions ifNil: [ (selector keywords collect: [:char| 0 ]) asIntegerArray ]
+	^keywordsPositions ifNil: [ (self selector keywords collect: [:char| 0 ]) asIntegerArray ]
```

Comment:
I imagine that we could rewrite it as 
IntegerArray new: selector keywords size withAll: 0